src/scripts/main.js

```diff
@@ -1,3 +1,37 @@
 'use strict';
 
-// write code here
+const listSalary = [...document.querySelectorAll('li')];
+
+function parseSalary(salaryStr) {
+  return parseInt(salaryStr.replace(/[$,]/g, ''), 10);
+}
```

Comment:
The function 'parseSalary' could use a comment explaining what it does. It's not immediately clear from the function name and the code inside it.